spec/lucky_cli/runner_spec.cr

```diff
@@ -21,6 +21,30 @@ describe LuckyCli::Runner do
       .should have_called_my_cool_task
   end
 
+  it "prints the help_message for a found task when a help flag is passed" do
+    LuckyCli::Runner
+      .run(args: ["my.cool_task", "--help"])
+      .should have_default_help_message
+    LuckyCli::Runner
+      .run(args: ["my.cool_task", "-h"])
+      .should have_default_help_message
+    LuckyCli::Runner
+      .run(args: ["my.cool_task", "help"])
+      .should have_default_help_message
```

Comment:
Maybe to make it clear what each of these tests is doing you could iterate over an array:

```crystal
%w( -h --help help).each do |help_arg|
  LuckyCli::Runner
     # ... run it and assert it
end
```